DontLeakActorsOnFailingConnectionSpecs fails on Windows

```
activeShells (actor: Actor[akka://DontLeakActorsOnFailingConnectionSpecs/user/StreamSupervisor-37/flow-0-0-unknown-operation#1262236023]):                                                                                                   
  GraphInterpreterShell                                                                                                                                                                                                                      
    GraphAssembly                                                                                                                                                                                                                            
      [ GraphStage(SingleSource(Range(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 
53, 54, 55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100))) [73c42cd9]    [Name(iterableSource), Name(
singleSource)]                                                                                                                                                                                                                               
        GraphStage(MapAsyncUnordered(4,<function1>)) [3ddc9b9b]    [Name(mapAsyncUnordered)]                                                                                                                                                 
        GraphStage(Map(<function1>)) [4df54276]    [Name(foreachSink), Name(foreachSink), Name(map)]                                                                                                                                         
        GraphStage(akka.stream.impl.fusing.GraphStages$IgnoreSink$@330e833a) [51ddacdf]    [Name(foreachSink), Name(foreachSink), Name(ignoreSink)]                                                                                          
        GraphStage(Map(<function1>)) [783114af]    [Name(map)]                                                                                                                                                                               
        GraphStage(StatefulMapConcat) [617ec366]    [Name(iterableSource), Name(statefulMapConcat)]                                                                                                                                          
      ]                                                                                                                                                                                                                                      
      [MapAsyncUnordered.in,Map.in,Ignore.in,Map.in,StatefulMapConcat.in]                                                                                                                                                                    
      [1,2,3,4,5]                                                                                                                                                                                                                            
      [Map.out,MapAsyncUnordered.out,Map.out,StatefulMapConcat.out,single.out]                                                                                                                                                               
      [4,1,2,5,0]digraph waits {                                                                                                                                                                                                             
N0 [label="SingleSource(Range(1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51, 52, 53, 54, 
55, 56, 57, 58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 100))"]                                                  
N1 [label="MapAsyncUnordered(4,<function1>)"]                                                                                                                                                                                                
N2 [label="Map(<function1>)"]                                                                                                                                                                                                                
N3 [label="akka.stream.impl.fusing.GraphStages$IgnoreSink$@330e833a"]                                                                                                                                                                        
N4 [label="Map(<function1>)"]                                                                                                                                                                                                                
N5 [label="StatefulMapConcat"]                                                                                                                                                                                                               
  N1 -> N4 [label=shouldPull; color=blue]                                                                                                                                                                                                    
  N1 -> N2 [label=shouldPush; color=red];                                                                                                                                                                                                    
  N2 -> N3 [label=shouldPush; color=red];                                                                                                                                                                                                    
  N4 -> N5 [label=shouldPull; color=blue]                                                                                                                                                                                                    
                                                                                                                                                                                                                                             
}                                                                                                                                                                                                                                            
// (8, 117, 117)() (running=5, shutdown=0,2,2,1,2,1)newShells:                                                                                                                                                                               
[info] DontLeakActorsOnFailingConnectionSpecs:                                                                                                                                                                                               
[info] Http.superPool                                                                                                                                                                                                                        
[info] - should not leak connection Actors when hitting non-existing endpoint *** FAILED ***                                                                                                                                                 
[info]   java.lang.AssertionError: assertion failed: expected no StreamSupervisor children, but got [Actor[akka://DontLeakActorsOnFailingConnectionSpecs/user/StreamSupervisor-37/flow-0-0-unknown-operation#1262236023]]                    
[info]   at scala.Predef$.assert(Predef.scala:170)                                                                                                                                                                                           
[info]   at akka.stream.testkit.Utils$$anonfun$assertAllStagesStopped$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(Utils.scala:31)                                                                                                                 
[info]   at akka.stream.testkit.Utils$$anonfun$assertAllStagesStopped$1$$anonfun$apply$mcV$sp$1.apply(Utils.scala:28)                                                                                                                        
[info]   at akka.stream.testkit.Utils$$anonfun$assertAllStagesStopped$1$$anonfun$apply$mcV$sp$1.apply(Utils.scala:28)                                                                                                                        
[info]   at akka.testkit.TestKitBase$class.poll$2(TestKit.scala:303)                                                                                                                                                                         
[info]   at akka.testkit.TestKitBase$class.awaitAssert(TestKit.scala:314)                                                                                                                                                                    
[info]   at akka.testkit.TestKit.awaitAssert(TestKit.scala:814)                                                                                                                                                                              
[info]   at akka.stream.testkit.Utils$$anonfun$assertAllStagesStopped$1.apply$mcV$sp(Utils.scala:28)                                                                                                                                         
[info]   at akka.stream.testkit.Utils$$anonfun$assertAllStagesStopped$1.apply(Utils.scala:26)                                                                                                                                                
[info]   at akka.stream.testkit.Utils$$anonfun$assertAllStagesStopped$1.apply(Utils.scala:26)                                                                                                                                                
[info]   ...                                                                                                                                                                                                                                 
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -315,7 +315,7 @@ There is a number of ways timeouts can be defined in Akka tests. The following w
 * `3.seconds` is third choice if not using testkit
 * lower timeouts must come with a very good reason (e.g. Awaiting on a known to be "already completed" `Future`)
 
-Special care should be given `expectNoMsg` calls, which indeed will wait the entire timeout before continuing, therefore a shorter timeout should be used in those, for example `200` or `300.millis`.
+Special care should be given `expectNoMessage` calls, which indeed will wait the entire timeout before continuing, therefore a shorter timeout should be used in those, for example `200` or `300.millis`.
 
 You can read up on remaining and friends in [TestKit.scala](https://github.com/akka/akka/blob/master/akka-testkit/src/main/scala/akka/testkit/TestKit.scala)
 

--- a/akka-http-core/src/test/scala/akka/http/impl/engine/client/HostConnectionPoolSpec.scala
+++ b/akka-http-core/src/test/scala/akka/http/impl/engine/client/HostConnectionPoolSpec.scala
@@ -425,7 +425,7 @@ class HostConnectionPoolSpec extends AkkaSpec(
           }
 
           def expectNoRequest(): Unit =
-            serverRequests.expectNoMsg()
+            serverRequests.expectNoMessage(remainingOrDefault)
 
           def pushResponse(response: HttpResponse = HttpResponse()) =
             serverResponses.sendNext(response)
@@ -483,7 +483,7 @@ class HostConnectionPoolSpec extends AkkaSpec(
           serverConnections.expectMsgType[ServerConnection]
 
         def expectNoNewConnection(): Unit =
-          serverConnections.expectNoMsg()
+          serverConnections.expectNoMessage(remainingOrDefault)
 
         protected override lazy val server =
           Flow.fromSinkAndSourceMat(

--- a/akka-http-core/src/test/scala/akka/http/impl/engine/client/HttpsProxyGraphStageSpec.scala
+++ b/akka-http-core/src/test/scala/akka/http/impl/engine/client/HttpsProxyGraphStageSpec.scala
@@ -29,7 +29,7 @@ class HttpsProxyGraphStageSpec extends AkkaSpec {
         flowOutProbe.sendNext(ByteString("HTTP/1.0 200 Connection established\r\n\r\n"))
 
         // we deliberately wait here some time to see if ProxyGraphStage properly handles slow demand from HTTP(S) proxy
-        flowInProbe.expectNoMsg(300.millis)
+        flowInProbe.expectNoMessage(300.millis)
 
         source.sendNext(ByteString("something"))
         flowInProbe.requestNext(ByteString("anything"))
@@ -61,7 +61,7 @@ class HttpsProxyGraphStageSpec extends AkkaSpec {
         flowOutProbe.sendNext(ByteString("HTTP/1.0 200 Connection established\r\n\r\n"))
 
         // we deliberately wait here some time to see if ProxyGraphStage properly handles slow demand from HTTP(S) proxy
-        flowInProbe.expectNoMsg(300.millis)
+        flowInProbe.expectNoMessage(300.millis)
 
         source.sendNext(ByteString("something"))
         flowInProbe.requestNext(ByteString("anything"))

--- a/akka-http-core/src/test/scala/akka/http/impl/engine/client/LowLevelOutgoingConnectionSpec.scala
+++ b/akka-http-core/src/test/scala/akka/http/impl/engine/client/LowLevelOutgoingConnectionSpec.scala
@@ -724,7 +724,7 @@ class LowLevelOutgoingConnectionSpec extends AkkaSpec("akka.loggers = []\n akka.
             |
             |""")
         netOutSub.request(1)
-        netOut.expectNoMsg(50.millis.dilated)
+        netOut.expectNoMessage(50.millis)
 
         sendWireData(
           """HTTP/1.1 100 Continue
@@ -761,7 +761,7 @@ class LowLevelOutgoingConnectionSpec extends AkkaSpec("akka.loggers = []\n akka.
             |
             |""")
         netOutSub.request(1)
-        netOut.expectNoMsg(50.millis.dilated)
+        netOut.expectNoMessage(50.millis)
 
         sendWireData(
           """HTTP/1.1 100 Continue
@@ -798,7 +798,7 @@ class LowLevelOutgoingConnectionSpec extends AkkaSpec("akka.loggers = []\n akka.
             |
             |""")
         netOutSub.request(1)
-        netOut.expectNoMsg(50.millis.dilated)
+        netOut.expectNoMessage(50.millis)
 
         sendWireData(
           """HTTP/1.1 200 OK
@@ -829,7 +829,7 @@ class LowLevelOutgoingConnectionSpec extends AkkaSpec("akka.loggers = []\n akka.
             |
             |""")
         netOutSub.request(1)
-        netOut.expectNoMsg(50.millis.dilated)
+        netOut.expectNoMessage(50.millis)
 
         sendWireData(
           """HTTP/1.1 400 Bad Request

--- a/akka-http-core/src/test/scala/akka/http/impl/engine/server/HttpServerSpec.scala
+++ b/akka-http-core/src/test/scala/akka/http/impl/engine/server/HttpServerSpec.scala
@@ -59,14 +59,14 @@ class HttpServerSpec extends AkkaSpec(
           data.to(Sink.fromSubscriber(dataProbe)).run()
           val sub = dataProbe.expectSubscription()
           sub.request(10)
-          dataProbe.expectNoMsg(50.millis.dilated)
+          dataProbe.expectNoMessage(50.millis)
 
           send("abcdef")
           dataProbe.expectNext(ByteString("abcdef"))
 
           send("ghijk")
           dataProbe.expectNext(ByteString("ghijk"))
-          dataProbe.expectNoMsg(50.millis.dilated)
+          dataProbe.expectNoMessage(50.millis)
       }
 
       shutdownBlueprint()
@@ -110,7 +110,7 @@ class HttpServerSpec extends AkkaSpec(
           val sub = dataProbe.expectSubscription()
           sub.request(10)
           dataProbe.expectNext(Chunk(ByteString("abcdef")))
-          dataProbe.expectNoMsg(50.millis.dilated)
+          dataProbe.expectNoMessage(50.millis)
 
           send("3ghi\r\n") // missing "\r\n" after the number of bytes
           val error = dataProbe.expectError()
@@ -169,7 +169,7 @@ class HttpServerSpec extends AkkaSpec(
 
           send("ghijk")
           dataProbe.expectNext(ByteString("ghijk"))
-          dataProbe.expectNoMsg(50.millis.dilated)
+          dataProbe.expectNoMessage(50.millis)
       }
 
       shutdownBlueprint()
@@ -194,7 +194,7 @@ class HttpServerSpec extends AkkaSpec(
 
           send("3\r\nghi\r\n")
           dataProbe.expectNext(Chunk(ByteString("ghi")))
-          dataProbe.expectNoMsg(50.millis.dilated)
+          dataProbe.expectNoMessage(50.millis)
       }
       shutdownBlueprint()
     })
@@ -283,7 +283,7 @@ class HttpServerSpec extends AkkaSpec(
 
           send("3\r\nghi\r\n")
           dataProbe.expectNext(ByteString("ghi"))
-          dataProbe.expectNoMsg(50.millis.dilated)
+          dataProbe.expectNoMessage(50.millis)
 
           send("0\r\n\r\n")
           dataProbe.expectNext(LastChunk)
@@ -341,7 +341,7 @@ class HttpServerSpec extends AkkaSpec(
           val sub = dataProbe.expectSubscription()
           sub.request(10)
           dataProbe.expectNext(Chunk(ByteString("abcdef")))
-          dataProbe.expectNoMsg(50.millis.dilated)
+          dataProbe.expectNoMessage(50.millis)
 
           send("0\r\n\r\n")
           dataProbe.expectNext(LastChunk)
@@ -412,7 +412,7 @@ class HttpServerSpec extends AkkaSpec(
           val sub = dataProbe.expectSubscription()
           sub.request(10)
           dataProbe.expectNext(ByteString("abcdef"))
-          dataProbe.expectNoMsg(50.millis.dilated)
+          dataProbe.expectNoMessage(50.millis)
           closeNetworkInput()
           dataProbe.expectError().getMessage shouldEqual "Entity stream truncation"
       }
@@ -434,7 +434,7 @@ class HttpServerSpec extends AkkaSpec(
           val sub = dataProbe.expectSubscription()
           sub.request(10)
           dataProbe.expectNext(Chunk(ByteString("abcdef")))
-          dataProbe.expectNoMsg(50.millis.dilated)
+          dataProbe.expectNoMessage(50.millis)
           closeNetworkInput()
           dataProbe.expectError().getMessage shouldEqual "Entity stream truncation"
       }
@@ -598,7 +598,7 @@ class HttpServerSpec extends AkkaSpec(
               |Date: XXXX
               |
               |""")
-          dataProbe.expectNoMsg(50.millis.dilated)
+          dataProbe.expectNoMessage(50.millis)
           send("0123456789ABCDEF")
           dataProbe.expectNext(ByteString("0123456789ABCDEF"))
           dataSub.request(1)
@@ -638,7 +638,7 @@ class HttpServerSpec extends AkkaSpec(
               |Date: XXXX
               |
               |""")
-          dataProbe.expectNoMsg(50.millis.dilated)
+          dataProbe.expectNoMessage(50.millis)
           send("""10
                  |0123456789ABCDEF
                  |0

--- a/akka-http-core/src/test/scala/akka/http/impl/engine/server/HttpServerTestSetupBase.scala
+++ b/akka-http-core/src/test/scala/akka/http/impl/engine/server/HttpServerTestSetupBase.scala
@@ -64,7 +64,7 @@ abstract class HttpServerTestSetupBase {
     }.mkString("\n")
 
   def expectRequest(): HttpRequest = requests.requestNext()
-  def expectNoRequest(max: FiniteDuration): Unit = requests.expectNoMsg(max)
+  def expectNoRequest(max: FiniteDuration): Unit = requests.expectNoMessage(max)
   def expectSubscribe(): Unit = netOut.expectComplete()
   def expectSubscribeAndNetworkClose(): Unit = netOut.expectSubscriptionAndComplete()
   def expectNetworkClose(): Unit = netOut.expectComplete()

--- a/akka-http-core/src/test/scala/akka/http/impl/engine/server/PrepareRequestsSpec.scala
+++ b/akka-http-core/src/test/scala/akka/http/impl/engine/server/PrepareRequestsSpec.scala
@@ -93,7 +93,7 @@ class PrepareRequestsSpec extends AkkaSpec {
       inSub.expectRequest(1)
 
       // bug would fail stream here with exception
-      upstreamProbe.expectNoMsg(100.millis.dilated)
+      upstreamProbe.expectNoMessage(100.millis)
 
       inSub.sendNext(ParserOutput.EntityChunk(HttpEntity.ChunkStreamPart(ByteString("abc"))))
       entityProbe.expectNext()

--- a/akka-http-core/src/test/scala/akka/http/impl/engine/ws/ByteStringSinkProbe.scala
+++ b/akka-http-core/src/test/scala/akka/http/impl/engine/ws/ByteStringSinkProbe.scala
@@ -9,6 +9,7 @@ import akka.actor.ActorSystem
 import akka.annotation.InternalApi
 import akka.stream.scaladsl.Sink
 import akka.stream.testkit.TestSubscriber
+import akka.testkit.TestKit
 import akka.util.ByteString
 
 import scala.annotation.tailrec
@@ -57,7 +58,7 @@ private[http] object ByteStringSinkProbe {
       }
       def expectNoBytes(timeout: FiniteDuration): Unit = {
         ensureRequested()
-        probe.expectNoMsg(timeout)
+        probe.expectNoMessage(timeout)
       }
 
       var inBuffer = ByteString.empty

--- a/akka-http-core/src/test/scala/akka/http/impl/engine/ws/MessageSpec.scala
+++ b/akka-http-core/src/test/scala/akka/http/impl/engine/ws/MessageSpec.scala
@@ -201,7 +201,7 @@ class MessageSpec extends FreeSpec with Matchers with WithMaterializerSpec with 
           parts.runWith(Sink.fromSubscriber(sub))
           val s = sub.expectSubscription()
           s.request(4)
-          sub.expectNoMsg(100.millis.dilated)
+          sub.expectNoMessage(100.millis)
 
           val header1 = frameHeader(Opcode.Continuation, data1.size, fin = true)
           pushInput(header1 ++ data1)
@@ -452,7 +452,7 @@ class MessageSpec extends FreeSpec with Matchers with WithMaterializerSpec with 
         pushInput(closeFrame(Protocol.CloseCodes.Regular, mask = true))
         expectComplete(messageIn)
 
-        netIn.expectNoMsg(100.millis.dilated) // especially the cancellation not yet
+        netIn.expectNoMessage(100.millis) // especially the cancellation not yet
         expectNoNetworkData()
         messageOut.sendComplete()
 
@@ -481,7 +481,7 @@ class MessageSpec extends FreeSpec with Matchers with WithMaterializerSpec with 
         pushInput(closeFrame(Protocol.CloseCodes.Regular, mask = true))
         expectComplete(messageIn)
 
-        netIn.expectNoMsg(100.millis.dilated) // especially the cancellation not yet
+        netIn.expectNoMessage(100.millis) // especially the cancellation not yet
         expectNoNetworkData()
         messageOut.sendComplete()
 

--- a/akka-http-core/src/test/scala/akka/http/impl/engine/ws/WebSocketIntegrationSpec.scala
+++ b/akka-http-core/src/test/scala/akka/http/impl/engine/ws/WebSocketIntegrationSpec.scala
@@ -49,7 +49,7 @@ class WebSocketIntegrationSpec extends AkkaSpec("akka.stream.materializer.debug.
       response.futureValue.response.status.isSuccess should ===(true)
       sink
         .request(10)
-        .expectNoMsg(500.millis.dilated)
+        .expectNoMessage(500.millis)
 
       source
         .sendNext(TextMessage("hello"))

--- a/akka-http-core/src/test/scala/akka/http/scaladsl/TightRequestTimeoutSpec.scala
+++ b/akka-http-core/src/test/scala/akka/http/scaladsl/TightRequestTimeoutSpec.scala
@@ -44,7 +44,7 @@ class TightRequestTimeoutSpec extends WordSpec with Matchers with BeforeAndAfter
       val response = Http().singleRequest(HttpRequest(uri = s"http://$hostname:$port/")).futureValue
       response.status should ===(StatusCodes.ServiceUnavailable) // the timeout response
 
-      p.expectNoMsg(1.second.dilated) // here the double push might happen
+      p.expectNoMessage(1.second) // here the double push might happen
 
       binding.flatMap(_.unbind()).futureValue
     }

--- a/akka-http-testkit/src/main/scala/akka/http/scaladsl/testkit/WSProbe.scala
+++ b/akka-http-testkit/src/main/scala/akka/http/scaladsl/testkit/WSProbe.scala
@@ -129,7 +129,7 @@ object WSProbe {
       }
 
       def expectNoMessage(): Unit = subscriber.expectNoMsg()
-      def expectNoMessage(max: FiniteDuration): Unit = subscriber.expectNoMsg(max)
+      def expectNoMessage(max: FiniteDuration): Unit = subscriber.expectNoMessage(max)
 
       def expectCompletion(): Unit = subscriber.expectComplete()
 

--- a/akka-http-tests/src/test/scala/akka/http/scaladsl/server/DontLeakActorsOnFailingConnectionSpecs.scala
+++ b/akka-http-tests/src/test/scala/akka/http/scaladsl/server/DontLeakActorsOnFailingConnectionSpecs.scala
@@ -33,7 +33,7 @@ abstract class DontLeakActorsOnFailingConnectionSpecs(poolImplementation: String
 
       http.host-connection-pool.pool-implementation = $poolImplementation
     }""").withFallback(ConfigFactory.load())
-  implicit val system = ActorSystem("DontLeakActorsOnFailingConnectionSpecs", config)
+  implicit val system = ActorSystem("DontLeakActorsOnFailingConnectionSpecs-" + poolImplementation, config)
   import system.dispatcher
   implicit val materializer = ActorMaterializer()
 
@@ -42,11 +42,12 @@ abstract class DontLeakActorsOnFailingConnectionSpecs(poolImplementation: String
   "Http.superPool" should {
 
     "not leak connection Actors when hitting non-existing endpoint" in {
+      val address = SocketUtil.temporaryServerAddress()
       assertAllStagesStopped {
         val reqsCount = 100
         val clientFlow = Http().superPool[Int]()
-        val host = "127.0.0.46" // unlikely to be used host / port
-        val port = 34763
+        val host = address.getHostString
+        val port = address.getPort
         val source = Source(1 to reqsCount)
           .map(i ⇒ HttpRequest(uri = Uri(s"http://$host:$port/test/$i")) → i)
 
@@ -61,7 +62,6 @@ abstract class DontLeakActorsOnFailingConnectionSpecs(poolImplementation: String
         countDown.await(10, TimeUnit.SECONDS) should be(true)
         Await.result(running, 10.seconds)
 
-        Thread.sleep(5000)
       }
     }
   }
@@ -69,11 +69,11 @@ abstract class DontLeakActorsOnFailingConnectionSpecs(poolImplementation: String
   private def handleResponse(httpResp: Try[HttpResponse], id: Int): Unit = {
     httpResp match {
       case Success(httpRes) ⇒
-        println(s"$id: OK: (${httpRes.status.intValue}")
+        system.log.info(s"$id: OK: (${httpRes.status.intValue}")
         httpRes.entity.dataBytes.runWith(Sink.ignore)
 
       case Failure(ex) ⇒
-        println(s"$id: FAIL: $ex")
+        system.log.error(ex, s"$id: FAIL")
     }
   }
 

--- a/akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala
+++ b/akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala
@@ -136,7 +136,7 @@ class Http2ServerSpec extends AkkaSpec("""
 
         sendHEADERS(1, endStream = true, endHeaders = false, fragment1)
         requestIn.ensureSubscription()
-        requestIn.expectNoMsg()
+        requestIn.expectNoMessage(remainingOrDefault)
         sendCONTINUATION(1, endHeaders = true, fragment2)
 
         val request = expectRequestRaw()
@@ -643,7 +643,7 @@ class Http2ServerSpec extends AkkaSpec("""
         val AckFlag = new ByteFlag(0x1)
         sendFrame(FrameType.PING, AckFlag, 0, ByteString("data1234"))
 
-        expectNoMsg(100 millis)
+        expectNoMessage(100.millis)
       }
       "respond to invalid (not 0x0 streamId) PING with GOAWAY (spec 6_7)" in new TestSetup with RequestResponseProbes {
         val invalidIdForPing = 1

--- a/docs/src/main/paradox/contributing.md
+++ b/docs/src/main/paradox/contributing.md
@@ -317,7 +317,7 @@ There is a number of ways timeouts can be defined in Akka tests. The following w
 * `3.seconds` is third choice if not using testkit
 * lower timeouts must come with a very good reason (e.g. Awaiting on a known to be "already completed" `Future`)
 
-Special care should be given `expectNoMsg` calls, which indeed will wait the entire timeout before continuing, therefore a shorter timeout should be used in those, for example `200` or `300.millis`.
+Special care should be given `expectNoMessage` calls, which indeed will wait the entire timeout before continuing, therefore a shorter timeout should be used in those, for example `200` or `300.millis`.
 
 You can read up on remaining and friends in [TestKit.scala](https://github.com/akka/akka/blob/master/akka-testkit/src/main/scala/akka/testkit/TestKit.scala)
 

